molgenis-core-ui/src/main/java/org/molgenis/core/ui/admin/permission/PermissionManagerController.java

```diff
@@ -107,11 +105,11 @@ public String init(Model model)
 
 	@PreAuthorize("hasAnyRole('ROLE_SU')")
 	@Transactional(readOnly = true)
-	@GetMapping("/entityclass/group/{groupId}")
+	@GetMapping("/entityclass/role/{roleName}")
```

Comment:
Can ```roleName``` contain non-alphanumeric characters? If so then {roleName:.+} is problably better here.

Reply:
Just things like MY_ROLE. Should be safe I think?